### PR TITLE
(MODULES-4370) Fix CI for Windows 2008

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

File: spec/support/utilities/iis_application_pool.rb [hunk 1]
```diff
@@ -1,15 +1,43 @@
 def has_app_pool(pool_name)
-  !(on(default, "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command \"& {Get-WebAppPoolState -Name #{@pool_name}}\"").stdout =~ /Started/i).nil?
+  command = []
+  command << "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command"
+  command << "\"& {"
```
Comment (glennsarti) [inline spec/support/utilities/iis_application_pool.rb]: Needs a space between command and `&`

File: spec/support/utilities/iis_application_pool.rb [hunk 2]
```diff
@@ -1,15 +1,43 @@
 def has_app_pool(pool_name)
-  !(on(default, "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command \"& {Get-WebAppPoolState -Name #{@pool_name}}\"").stdout =~ /Started/i).nil?
+  command = []
+  command << "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command"
```
Comment (glennsarti) [inline spec/support/utilities/iis_application_pool.rb]: Running old powershell via Cygwin has an annoying bug to with STDIN redirection

https://tickets.puppetlabs.com/browse/PE-18351?focusedCommentId=394366&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-394366

Fix was implemented here;
https://github.com/puppetlabs/pe_acceptance_tests/pull/1195

Simple as adding ` < /dev/null` to the end.

**Comment by glennsarti** [comment]
@jpogran Can we put these changes under MODULES-4370 instead of maint?

**Comment by jpogran** [comment]
Did not know MODULES-4370 existed. Will rename PR and modify commit message. This is also turning into a larger fix than a few simple edits so will split out commits.

**Comment by glennsarti** [comment]
Merged without successful Travis CI run due to Travis having errors due to S3 outage.  Also ran locally and via adhoc pipelines.

**Pull request merged**